test/e2e/ccm_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
 )
 
-var _ = Describe("Nutanix flavor CCM", func() {
+var _ = Describe("Nutanix flavor CCM", Label("capx-feature-test", "slow", "network"), func() {
```

Comment:
Please add a specific label so we can target the specific ccm tests

Reply:
added